input/metadata/properties/default.json

```
@@ -3482,7 +3482,7 @@
         "type": "number",
         "minimum": 0.87,
         "maximum": 1.04,
-        "default": 0
+        "default": 0.87
```

```suggestion
        "default": 0
```
restore the original value

---

This one is an incorrect default which falls outside the minimum-maximum range. I confirmed the new value with Joe but will document this in the new PR for validating properties.